persist/sqlite/wallet.go

```diff
@@ -689,11 +693,13 @@ SELECT
 	ev.date_created, 
 	ci.height, 
 	ci.block_id, 
+	last_chain_index.height - ci.height AS confirmations,
```

Comment:
do we want to handle the case where `last_chain_index.height` is lower than `ci.height`?

Reply:
That shouldn’t be possible